tutorials/3_RunningBundleOfRuns.md

```diff
@@ -64,7 +64,7 @@ Rscript start.R --test config/scenario_config_XYZ.csv
 ```
 Running the complete chain of runs, but only for one region and one iteration (as long as `cm_iteration_max = 1` wasn't changed), can be started with:
```

Comment:
you can delete "(as long as `cm_iteration_max = 1` wasn't changed)" because it is set to 1 always with `-q`

Reply:
done, thanks!